_posts/2022-03-02-getting-started-project-flotta.markdown

```diff
@@ -41,6 +41,11 @@ For kubernetes:
 $ kubectl apply -f https://github.com/project-flotta/flotta-operator/releases/download/v0.0.1/k8s-flotta-operator.yaml
 ```
 
+The only difference between the two files is in how [OCP and Kubernetes](https://cloud.redhat.com/blog/kubernetes-ingress-vs-openshift-route) enable external access to services.
```

Comment:
There is additional different which is creating monitoring resources only for OCP since for k8s it requires prometheus to be enabled on the cluster (while it is already for OCP).
But enabling the monitoring stack for k8s needs to done not via the operator.yaml. There it is excluded from it.